modules/partition-replicator/src/test/java/org/apache/ignite/internal/partition/replicator/PartitionReplicaLifecycleManagerTest.java

```diff
@@ -146,13 +151,16 @@ void setUp(
 
         when(distributionZoneManager.dataNodes(anyLong(), anyInt(), anyInt())).thenReturn(completedFuture(Set.of(nodeName)));
 
-        when(zoneResourcesManager.allocateZonePartitionResources(any(), anyInt()))
+        when(zoneResourcesManager.allocateZonePartitionResources(any(), anyInt(), any()))
                 .thenReturn(new ZonePartitionResources(
                         txStatePartitionStorage,
                         raftGroupListener,
                         partitionSnapshotStorageFactory
                 ));
 
+        when(raftManager.startRaftGroupNode(any(), any(), any(), any(), (RaftGroupOptions) any(), any()))
```

Comment:
why do you need the cast to `RaftGroupOptions`?

Reply:
In order to resolve ambiguous method call.

Reply:
Please use `any(RaftGroupOptions.class)` then